posts/stay_calm_stop_mocking.md

```diff
@@ -0,0 +1,551 @@
+<meta name="id" content="5059786785061561365">
```

Comment:
@alechenninger Still reviewing this, but do you think there is value in doing a before and after in code? That's how I embraced this concept in the first place. We were trying to implement a test utility for ProductService that could be reused across different tests. But it was using Mockito and we switched it out to use Fakes. And it was like the fog suddenly cleared.